lib/data_pull.rb

```diff
@@ -91,32 +98,44 @@ def render_output(rows)
 
   # @api private
   # A subtask is a class that has a run method, the type signature should look like:
-  # +#run(args: Array<String>, include_missing: Boolean) -> Result+
+  # +#run(args: Array<String>, config: Config) -> Result+
   # @return [Class,nil]
   def subtask(name)
     {
       'uuid-lookup' => UuidLookup,
       'uuid-convert' => UuidConvert,
       'email-lookup' => EmailLookup,
+      'ig-request' => InspectorGeneralRequest,
```

Comment:
Petty nit: the existing naming convention seems to be things you can look up about a user/account -- their uuid, their email, but not their Inspector General. Would this make more sense as `shared-devices` or something? It's actually not abundantly clear to me what this is for.

That said, that may be the point?

Reply:
The inspector general is a GSA office, so we're running the standard report with the attributes that the GSA office asks for. And the final export of this includes more than just their devices so I feel like that name might sell it short